vision/keypoints/fastkp.go

```diff
@@ -22,10 +22,10 @@ import (
 // FASTConfig holds the parameters necessary to compute the FAST keypoints.
 type FASTConfig struct {
 	NMatchesCircle int  `json:"n_matches"`
-	NMSWinSize     int  `json:"nms_win_size"`
-	Threshold      int  `json:"threshold"`
-	Oriented       bool `json:"oriented"`
-	Radius         int  `json:"radius"`
+	NMSWinSize     int  `json:"nms_win_size"` // TODO(RSDK-586): unit (px)?
+	Threshold      int  `json:"threshold"`    // TODO(RSDK-586): unit (px)?
+	Oriented       bool `json:"oriented"`     // TODO(RSDK-586): unit (px)?
+	Radius         int  `json:"radius"`       // TODO(RSDK-586): unit (px)?
```

Comment:
NMSWinSize has units of pixels
threshold is a value between 0 and 255, and represents a unit of grayscale intensity of the pixel
oriented is true/false
radius is in pixels

Reply:
just gonna add a comment for `threshold is a value between 0 and 255, and represents a unit of grayscale`

and oriented needs no unit.... sorry i likely copypasta'd the TODO